cassandra/README.md

```diff
@@ -65,6 +65,45 @@ Docker Hub.
 <!-- Optional -->
 ### Deploy with Kubernetes
 
+This image can also be deployed on a Kubernetes cluster, such as [minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).The following example YAML files are provided in the repository as reference for Kubernetes deployment:
+
+- [`cassandra-deployment.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/cassandra/cassandra-deployment.yaml): example using default configuration to create a basic cassandra service.
+
+- [`cassandra-deployment-conf.yaml`](https://github.com/clearlinux/dockerfiles/blob/master/cassandra/cassandra-deployment-conf.yaml): example using your own custom configuration to create a cassandra service.
+
+  
+
+Steps to deploy cassandra on a Kubernetes cluster:
+
+1. If you want to deploy `cassandra-deployment.yaml`
+
+   ```
+   kubectl create -f cassandra-deployment.yaml
+   ```
+
+   Or if you want to deploy `cassandra-deployment-conf.yaml`  
+
+   ```
+   kubectl create -f cassandra-deployment-conf.yaml
+   ```
+
+2. Install cassandra bundle and add environmental variable.
+
+   ```
+   swupd bundle-add cassandra
```

Comment:
Please add more description to explain what's purpose for these codes. And, please try if we can add these into a file and make it easily for user.

Reply:
Ok, I will fixed it soon!